test/benchmark/preload.lua

```diff
@@ -1,4 +1,4 @@
 -- Modules loaded here will not be cleared and reloaded by Busted.
 -- Busted started doing this to help provide more isolation.  See issue #62
 -- for more information about this.
-local t = require('test.functional.testutil')
+local n = require('test.functional.nvimsession')()
```

Comment:
Calling the returned function here doesn't do anything.
```suggestion
local n = require('test.functional.nvimsession')
```